tasks/Ubuntu.yml

```diff
@@ -2,24 +2,29 @@
 # tasks file for osquery (Ubuntu specific)
 
 - name: Download osquery APT key
-  become: yes
   apt_key:
     keyserver: keyserver.ubuntu.com
     id: C9D8B80B
     state: present
   tags:
     - osquery
 
+- name: Ensure apt-transport-https is installed
+  apt:
+    name: apt-transport-https
+    state: present
+  when: ansible_distribution_release == 'xenial'
+  tags:
+    - osquery
+
 - name: Configure osquery APT repository
-  become: yes
   apt_repository:
-    repo: 'deb [arch=amd64] https://osquery-packages.s3.amazonaws.com/trusty trusty main'
+    repo: 'deb [arch=amd64] https://osquery-packages.s3.amazonaws.com/{{ ansible_distribution_release }} {{ ansible_distribution_release }} main'
```

Comment:
I would recommend using the 'lower' filter.

`ansible_distribution_release|lower`